.github/workflows/SQL2019.yaml

```diff
@@ -0,0 +1,168 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: BuildAndTest
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+#env:
+#  USER: ${{ secrets.USER }}
+#  PASSWORD: ${{ secrets.PASSWORD }}
+#  PASSWORD_AIRLINE_USER: ${{ secrets.PASSWORD_AIRLINE_USER }}
+#  PASSWORD_AIRLINE_USER_DBOWNER: ${{ secrets.PASSWORD_AIRLINE_USER_DBOWNER }}
+#  DRIVER: "{ODBC Driver 17 for SQL Server}"
+
+jobs:
+  SQL19:
+    runs-on: windows-2022
+    
+    env:
+      # Define CI to skip some test case.
+      CI: True
+      r-version: "3.5.2"
+      python-version: "3.7.1"
+      sql-platform: "box"
+
+
+    defaults:
+      run:
+        shell: cmd
+
+    steps:
+
+      - name: Generate password
+        run: |
+          setlocal enabledelayedexpansion
+          set password=%random%%random%%random%qaZ~@
+          echo dbPassword=!password! >> %GITHUB_ENV%
+
+      # Downloads the R cabinet file for the initial release version of SQL Server 2019.
+      - name: Download Microsoft R Open cabinet file
+        run: |
+          set CurrentDir=%cd%
+          curl -L https://go.microsoft.com/fwlink/?linkid=2085686 -o "%CurrentDir%/SRO_3.5.2.125_1033.cab"
+        shell: cmd
+      
+      # Downloads the Python cabinet file for the initial release  version of SQL Server 2019.
+      - name: Download Microsoft Python Open cabinet file
+        run: |
+          set CurrentDir=%cd%
+          curl -L https://go.microsoft.com/fwlink/?linkid=2085793 -o "%CurrentDir%/SPO_4.5.12.120_1033.cab"
+        shell: cmd
+
+      - name: Install SQL Server 2019
```

Comment:
good to have comment that explains why you chose certain install params -> 
- /SAPWD=%dbPassword% /SECURITYMODE=SQL /TCPENABLED=1 for accessing db without windows account and using localhost address
- MLS params: ...